tests/test_pi_io.py

```diff
@@ -39,5 +39,3 @@ def test_setup_only_called_once_per_pin(self):
         self.assertEqual(mock_GPIO.setup.call_args_list,
```

Comment:
Oh, I didn't notice before but `IO` should be `io` otherwise it looks like a class:
https://google.github.io/styleguide/pyguide.html?showone=Naming#Naming